lib/rex/parser/arguments.rb

```diff
@@ -70,10 +104,31 @@ def parse(args, &_block)
       def usage
         txt = ["\nOPTIONS:\n"]
 
-        fmt.sort.each do |entry|
+        sorted_fmt = fmt.sort_by do |element|
```

Comment:
I know you're just sorting alphabetically here because that's what was done before but do we need to? maybe there's a good reason for it but I think I would prefer the displayed order to be what's defined in the code that way similar items (e.g. sessions -k and -K) could be grouped together